src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let result = days * 40;
+  const lowDiscount = 20;
+  const highDiscount = 50;
```

Comment:
<img width="450" alt="Screenshot 2023-11-30 at 14 33 24" src="https://github.com/mate-academy/js_task-transportation-on-vacation/assets/135232001/a1ff0182-18a7-4b53-b5b6-12fc3ae48f40">
https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md Consider using the checklist for this task to name constants properly. There are a bunch of rules for constant naming. 
<img width="430" alt="Screenshot 2023-11-30 at 14 35 40" src="https://github.com/mate-academy/js_task-transportation-on-vacation/assets/135232001/bf30764a-dada-45f9-8085-23d17ee6ba7e">

Reply:
That what you mean?

Reply:
![image](https://github.com/mate-academy/js_task-transportation-on-vacation/assets/138024608/8dd64bea-7406-4b59-a381-e7e3b67c6fed)
btw at video he is using as well const and I used same as he does